native_client/java/standalone.md

```diff
@@ -0,0 +1,193 @@
+# DeepSpeech for standalone Java
+
+### Adding DeepSpeech to your Java project
+>Note: You can do this on your own way too, but this works with the example usage code provided below.
+1. Copy `libdeepspeech.jar` from `./build/` into `{YOUR PROJECT ROOT}/libs/`

```

Comment:
please make sure we don't push people to re-building code themselves except when really required, `.jar` should have everything bundled as well and people should pull from repos.